### PR TITLE
add random + some minor bugfixes

### PR DESCRIPTION
**Pull request by anakwaboe4**

### CODE REVIEW

File: backend/TikTakToe.Engines/Engines/random/random.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                if(Domove(board, i) != 0)
+                {
+                    legalMoves.Add(i);
+                }
+;
+            }
+
+            int index = randomPicker.Next(legalMoves.Count);
+            return legalMoves[index];
+
```
Comment (EwoutHeb) [inline backend/TikTakToe.Engines/Engines/random/random.cs]: remove empty lines

File: backend/TikTakToe.Engines/Engines/random/random.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                if(Domove(board, i) != 0)
+                {
+                    legalMoves.Add(i);
+                }
+;
+            }
+            
+            int index = randomPicker.Next(legalMoves.Count);
+            return legalMoves[index];
+
```
Comment (EwoutHeb) [inline backend/TikTakToe.Engines/Engines/random/random.cs]: remove empty line

File: backend/TikTakToe.Engines/Engines/random/random.cs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        private Random randomPicker = new Random();
+
+        public double Bench(int depth)
+        {
+            sw.Restart();
+            sw.Start();
+            SetPos(0);
+            sw.Stop();
+            return sw.ElapsedMilliseconds;
+
```
Comment (EwoutHeb) [inline backend/TikTakToe.Engines/Engines/random/random.cs]: remove empty line

File: backend/TikTakToe.Engines/Engines/random/random.cs [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        {
+            board = Domove(board, move);
+            List<int> legalMoves = new List<int>();
+            for(int i = 1; i <= 9; i++)
+            {
+                if(Domove(board, i) != 0)
+                {
+                    legalMoves.Add(i);
+                }
+;
```
Comment (EwoutHeb) [inline backend/TikTakToe.Engines/Engines/random/random.cs]: lost ";"?

File: backend/TikTakToe.Engines/Engines/random/random.cs [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        {
+            board = position;
+            List<int> legalMoves = new List<int>();
+            for(int i = 1; i <= 9; i++)
+            {
+                if(Domove(board, i) != 0)
+                {
+                    legalMoves.Add(i);
+                }
+;
```
Comment (EwoutHeb) [inline backend/TikTakToe.Engines/Engines/random/random.cs]: lost ";"?

File: backend/TikTakToe.Engines/Engines/random/random.cs (hunk 4 above)
Comment (anakwaboe4) [reply backend/TikTakToe.Engines/Engines/random/random.cs]: don't know how this even build succefully :(

**Comment by anakwaboe4** [comment]
@EwoutHeb can you please revieuw again, at least a few more bugs have been fixed since this PR

File: backend/TikTakToe.Engines/Engines/random/random.cs [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        public Core.Enums.Engines Engine => Core.Enums.Engines.Random;
+
+        private int board;
+        private Stopwatch sw = new Stopwatch();
+        private Random randomPicker = new Random();
+
+        public double Bench(int depth)
+        {
+            sw.Restart();
+            sw.Start();
```
Comment (EwoutHeb) [inline backend/TikTakToe.Engines/Engines/random/random.cs:22]: restart and start at the same time?

**Review by EwoutHeb — APPROVED** [review]
looks good, only one comment, but I assume this works

File: backend/TikTakToe.Engines/Engines/random/random.cs (hunk 6 above)
Comment (anakwaboe4) [reply backend/TikTakToe.Engines/Engines/random/random.cs:22]: it work, but has been changed anyway as discussed before

**Pull request merged**